app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ReservationsController < ApplicationController
+  def index
+    @reservations = Reservation.all.where("day >= ?", Date.current).where("day < ?", Date.current >> 3).order(day: :desc)
+  end
+
+  def new
+    @reservation = Reservation.new
+    @day = params[:day]
+    @time = params[:time]
+    @start_time = DateTime.parse(@day + " " + @time + " " + "JST")
```

Comment:
ここに限らず基本的に文字列連結は式展開使った方が nil の場合勝手に空白にしてくれたりするので↓のほうが良さそうと思いました！

```suggestion
    @start_time = DateTime.parse("#{@day} #{@time} JST")
```